frontend/src/metabase/actions/components/ActionViz/ActionVizForm.tsx

```diff
@@ -59,6 +59,12 @@ function ActionVizForm({
   const [showEditModal, setShowEditModal] = useState(false);
   const title = getFormTitle(action);
 
+  // store values without rerender
+  const formValuesRef = useRef<ParametersForActionExecution | undefined>();
```

Comment:
This seems not to be needed since the `ActionForm` is still mounted when the action is being edited in another modal. And this is actually good since it allows us to simplify the implementation. We don't need to store the values here, we need to make the action form respond to `Action` entity being updated. Does it make sense?